.github/workflows/build.yml

```
@@ -15,23 +16,24 @@ jobs:
       fail-fast: false
       matrix:
         binary: [
+          "binutils",
           "bison",
           "gawk",
           "gcc",
           "make",
           "python3",
-          "sed",]
+          "sed"]
```

```suggestion
          "sed",
          ]
```
if it's valid syntax